social-messenger-ts/src/components/MessagingChannelList/MessagingChannelList.css

```diff
@@ -1,7 +1,10 @@
+.messaging__sidebar {
+  height: calc(100% - 110px);
```

Comment:
Why the calc here?

Reply:
Updated to display grid. Also noticed a need for background.